odoo/addons/base/migrations/13.0.1.3/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,798 @@
+# NOTHING TO DO
+
+base         / ir.model.fields.selection / field_id (many2one)           : NEW relation: ir.model.fields, required
+base         / ir.model.fields.selection / name (char)                   : NEW required
+base         / ir.model.fields.selection / sequence (integer)            : NEW hasdefault
+base         / ir.model.fields.selection / value (char)                  : NEW required
+# NOTHING TO DO: new models
+
+base         / ir.model.relation        / date_init (datetime)          : DEL
+base         / ir.model.relation        / date_update (datetime)        : DEL
+# NOTHING TO DO
+
+base         / ir.module.module.dependency / auto_install_required (boolean): NEW hasdefault
+# NOTHING TO DO
+
+base         / ir.server.object.lines   / evaluation_type (selection)   : NEW required, selection_keys: ['equation', 'reference', 'value'], req_default: function, hasdefault
+base         / ir.server.object.lines   / type (selection)              : DEL required, selection_keys: ['equation', 'reference', 'value'], req_default: function
+# NOTHING TO DO
+
+base         / ir.translation           / type (selection)              : selection_keys is now '['code', 'model', 'model_terms']' ('['code', 'constraint', 'model', 'model_terms', 'selection', 'sql_constraint']')
+# NOTHING TO DO
+
+base         / ir.ui.view               / arch_prev (text)              : NEW
+base         / ir.ui.view               / arch_updated (boolean)        : NEW
+# NOTHING TO DO
+
+base         / report.layout            / __last_update (datetime)      : previously in module web
+base         / report.layout            / display_name (char)           : previously in module web
+base         / report.layout            / image (char)                  : previously in module web
+base         / report.layout            / name (char)                   : NEW
+base         / report.layout            / pdf (char)                    : previously in module web
+base         / report.layout            / view_id (many2one)            : previously in module web
+# NOTHING TO DO
+
+base         / res.company              / favicon (binary)              : NEW attachment: True, hasdefault
+base         / res.company              / font (selection)              : NEW selection_keys: ['Lato', 'Montserrat', 'Open_Sans', 'Oswald', 'Raleway', 'Roboto'], hasdefault
+base         / res.company              / primary_color (char)          : NEW
+base         / res.company              / secondary_color (char)        : NEW
+# NOTHING TO DO
+
+base         / res.lang                 / translatable (boolean)        : DEL
+# NOTHING TO DO
+
+base         / res.lang                 / url_code (char)               : NEW required
+# Done: post-migration
+
+base         / res.lang                 / week_start (selection)        : selection_keys is now '['1', '2', '3', '4', '5', '6', '7']' ('[1, 2, 3, 4, 5, 6, 7]')
```

Comment:
This implies a column conversion from INTEGER to VARCHAR. Does fields.Selection support this conversion?

Reply:
I fixed it.